build/teamcity/internal/release/process/make-and-publish-build-tagging.sh

```diff
@@ -55,6 +55,10 @@ if [[ -n "${release_branch}" ]] ; then
   log_into_gcloud
   gcloud container images add-tag "${gcr_repository}:${build_name}" "${gcr_repository}:latest-${release_branch}-build"
 fi
+if [[ "$TC_BUILD_BRANCH" == "master" ]]; then
+  log_into_gcloud
+  gcloud container images add-tag "${gcr_repository}:${build_name}" "${gcr_repository}:latest-master-build"
```

Comment:
Don't we also need to remove the label from the previous push? There should always be at most one image with this tag, right?

Reply:
AFAIK, this push replaces all previously published tags with the same name. This is the same logic we use for `latest-v23.1-build` tag.

Reply:
Indeed. It's a property of container repositories that a given tag can be on only one image in a repository at a time.

Reply:
Thanks! I think I was regressing to this incantation which we had to use for updating the "latest" label after a new AMI was created,

```
gcloud compute images remove-labels $IMAGE --labels="latest"`
```